java-samples/corda5-negotiation-cordapp/workflows/src/main/java/com/r3/developers/samples/negotiation/workflows/util/ListProposal.java

```diff
@@ -0,0 +1,59 @@
+package com.r3.developers.samples.negotiation.workflows.util;
+
+import com.r3.developers.samples.negotiation.states.Proposal;
+import net.corda.v5.application.flows.ClientRequestBody;
+import net.corda.v5.application.flows.ClientStartableFlow;
+import net.corda.v5.application.flows.CordaInject;
+import net.corda.v5.application.marshalling.JsonMarshallingService;
+import com.r3.developers.samples.negotiation.workflows.util.ListProposalResults;
```

Comment:
this line looks like an unused import. can you try removing this line and seeing if everything still works?

i was able to build the vnodes without the unused imports, and run listproposal command used the readme body.

if it works, remove all the unused imports from all other files

Reply:
Removed them and there were errors so those imports are being used